runtime/reference/cli/bench.md

```diff
@@ -114,13 +114,12 @@ Deno.bench("foo", async (b) => {
 
   // End measurement here.
   b.end();
-
-  // Now we can perform some potentially time-consuming teardown that will not
-  // taint out benchmark results.
-  file.close();
```

Comment:
Probably this becomes an error at some point when we migrated from Reader/Writer pattern to Web Stream API in the above code (The above web stream call closes the file automatically when it's read)